recipes/tbb/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "2019_u9":
     folder: all
+  "2020.0":
```

Comment:
the versioning of this lib looks insane to me... 😫

Reply:
I know, and I agree. They described the new release format here: https://github.com/intel/tbb/releases/tag/v2020.0